src/applications/app-errors.js

```diff
@@ -89,15 +91,7 @@ export function transformErr(ogErr, appOrParcel) {
     }
   }
 
-  result.appName = appOrParcel.name;
-  result.appOrParcelName = appOrParcel.name;
-  try {
-    result.name = appOrParcel.name;
-  } catch (err) {
-    // See https://github.com/CanopyTax/single-spa/issues/323
```

Comment:
This issue explains why we even have so many different properties for the same thing